### DISCUSSION TITLE
kangaroo/hosted/operator/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1381] [Announcements]
# kangaroo/hosted/operator/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kangaroo/hosted/operator/

**Comment by gssjl2008** [comment]
完全无法跟着操作

**Comment by windsonsea** [comment]
ui 变动较大，正在更新文档，感谢反馈🙏

**Comment by ModetaNiu** [comment]
PR #1601 更新了版面
